tests/conftest.py

```diff
@@ -6,6 +6,10 @@
 
 if TYPE_CHECKING:
     from ontopy.ontology import Ontology
+    from typing import Sequence
+import sys, os
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "utilities"))
```

Comment:
Would be nice with a comment explaining the purpose of this?

Reply:
Done